apps/feedahelp/src/components/ui/features/hero/data.ts

```diff
@@ -0,0 +1,84 @@
+export const items = [
+  {
+    imgSrc: "https://source.unsplash.com/ezSFnAFi9hY/500x500",
```

Comment:
Take few images from here:
https://drive.google.com/drive/folders/14xwGSmw5BJOPXyKcPwydIEZ2bulU6Z0O?usp=drive_link, add them in `public` and take it from there
we will later have CMS ticket for this

Reply:
done